Guides/dev-set-up-environment.md

```
@@ -8,255 +8,424 @@
-
- - **Code Editor**:
-    - Install a code editor of your preference (e.g. [Visual Studio Code](https://code.visualstudio.com/), [Eclipse](https://www.eclipse.org/), [Sublime Text](https://www.sublimetext.com/), [Vim](https://www.vim.org/)/[Neovim](https://neovim.io/) (RIP [Atom](https://github.blog/2022-06-08-sunsetting-atom/))).
-
- - **Docker**:
-    - Docker is useful if you do not have access to a persistent development environment that hosts the core system microservices that you need to test your integrations. With Docker, you can deploy containerized microservices on your local machine. Follow the instructions on the official Docker website to [install Docker Desktop on Windows](https://docs.docker.com/desktop/install/windows-install/). Remember that Docker Desktop on Windows also requires Linux on Windows that you can [install with WSL](https://docs.docker.com/desktop/install/windows-install/)
+    - In our deployment, Git is mostly installed with default options, with the following specific exceptions:
+      - Select VS Code as the default editor
+      - Check-out as-is, commit as-is
+      - Use Windows's default console window
 
   - **Postman / Newman**
```

markdownlint-cli2 (0.20.0)

[warning] Inconsistent indentation for list items at the same level
Expected: 1; Actual: 2

(MD005, list-indent)